tromega/lib/widgets/account/questions_widget.dart

```diff
@@ -59,13 +63,37 @@ class _SecondQuestionWidget extends State<QuestionWidget> {
           crossAxisAlignment: CrossAxisAlignment.stretch,
           children: [
             buildQuestion(text: 'Wie heißt du?'),
-            AnswerFieldWidget(controller: changedName),
-            buildQuestion(text: 'Wie alt bist du?'),
-            buildTextField(color),
+            AnswerFieldWidget(
+              maxLength: 30,
+              controller: changedName,
+              hintText: "Vorname Nachname",
+              suffixText: "",
+              regExp: r'^[a-zA-Z ]+$',
+            ),
+            buildQuestion(text: 'Wann hast du Geburtstag?'),
+            AnswerFieldWidget(
+              maxLength: 10,
+              controller: changedBirthday,
+              hintText: "YYYY-MM-DD",
+              suffixText: "",
+              regExp: r'^[\d-]+$',
+            ),
             buildQuestion(text: 'Wie groß bist du?'),
-            AnswerFieldWidget(controller: changedHeight),
+            AnswerFieldWidget(
+              maxLength: 3,
+              controller: changedHeight,
+              hintText: "0",
+              suffixText: "cm",
```

Comment:
bitte einheitlich. wenn ich hier 182cm groß bin und in der anderen view 182m, dann hat tromega eine kranke transformation ermöglicht